config/routes.rb

```diff
@@ -6,6 +6,8 @@
   
   get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
   get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
+  post '/users/:user_id/posts/:id/comments', to: 'posts#createcomment', as: 'create_user_post_comment'
+  post '/users/:user_id/posts', to: 'posts#create', as: 'create_user_post'
   get '/users', to: 'users#index', as: 'users'
   get '/users/:id', to: 'users#show', as: 'user'
 
```

Comment:
- [ ] Kindly update the routes for `likes & comments` accordingly.

Reply:
@Deepakdanger thanks I am still a learner that's why I am getting more confused as in the project requirements there's nowhere we need to apply the likes functionality can I skip it again I am not understand it how I can update my routes please you mean to rearrange it